drivers/lte_link_control/lte_lc.c

```diff
@@ -11,22 +11,39 @@
 #include <string.h>
 #include <stdio.h>
 #include <device.h>
+#include <lte_lc.h>
 #include <at_cmd.h>
 #include <at_cmd_parser/at_cmd_parser.h>
 #include <at_cmd_parser/at_params.h>
 #include <logging/log.h>
 
 LOG_MODULE_REGISTER(lte_lc, CONFIG_LTE_LINK_CONTROL_LOG_LEVEL);
 
-#define LC_MAX_READ_LENGTH 128
-#define AT_CMD_SIZE(x) (sizeof(x) - 1)
+#define LC_MAX_READ_LENGTH		128
+#define AT_CMD_SIZE(x)			(sizeof(x) - 1)
+#define AT_CEREG_5			"AT+CEREG=5"
+#define AT_CEREG_READ			"AT+CEREG?"
+#define AT_CEREG_PARAMS_COUNT		10
+#define AT_CEREG_ACTIVE_TIME_INDEX	8
+#define AT_CEREG_TAU_INDEX		9
+#define AT_CEREG_RESPONSE_MAX_LEN	80
+
+/* Lookup table for T3324 timer used for PSM active time. Unit is seconds.
+ * Ref: GPRS Timer 2 IE in 3GPP TS 24.008 Table 10.5.163/3GPP TS 24.008.
+ */
+static u32_t t3324_lookup[8] = {2, 60, 600, 0, 0, 0, 0, 0};
```

Comment:
Any reason why this is not const? Same goes for t3412_lookup.

Reply:
No sensible reason, fixed